webClient/src/app/editor/code-editor/monaco-settings/monaco-settings.component.html

```diff
@@ -10,27 +10,27 @@
   -->
 <div class="outer">
   <div class="container">
-    <h2>Editor Settings</h2>
+    <h4 class="title-card title-card-first">Basic</h4>
```

Comment:
Basic... what? This loses info and has me the user confused.

Reply:
![image](https://user-images.githubusercontent.com/20528015/108117958-a95a2180-706b-11eb-9b3d-325b66e5577f.png)


we see some redundancy is my thinking